ee/server/service/software_installers.go

```diff
@@ -391,7 +393,7 @@ func (svc *Service) BatchSetSoftwareInstallers(ctx context.Context, tmName strin
 
 			resp, err := client.Do(req)
 			if err != nil {
-				if errors.Is(err, fleethttp.ErrMaxSizeExceeded) {
+				if errors.Is(err, fleethttp.ErrMaxSizeExceeded) || errors.Is(err, &http.MaxBytesError{}) {
```

Comment:
Ah sorry, `errors.Is` is like `err == <the value>` somewhere in its chain of wrapped errors, it won't work for this. You want `errors.As` (which is "is an err in the chain assignable to this type").

Reply:
I'm dumb, thank you!